data/lib/src/network/linshare_http_client.dart

```diff
@@ -734,11 +737,24 @@ class LinShareHttpClient {
   }
 
   Future<String> getOIDCConfiguration(Uri baseUrl) async {
-    final result = await _dioClient.get(
-        Endpoint.oidcConfiguration.generateOIDCConfigurationUrl(baseUrl));
+    final result = await _dioClient.get(Endpoint.oidcConfiguration.generateBaseUrl(baseUrl));
     return result;
   }
 
+  Future<SecretTokenResponse> getSecretToken(Uri baseUrl, PlanBodyRequest planBodyRequest) async {
+    final result = await _dioClient.post(
```

Comment:
we can use the same _dioClient for getting `secretToken` from other url

Reply:
Of course !